README.md

```diff
@@ -443,59 +449,72 @@ The handler is expected to call its callback once with no parameters (failing to
 function setConfigurationHandler(newHandler)
 ```
 ###### Description
-Sets the new user handler for the configuration updates. This handler will be called every time a new configuration is created or an old configuration is updated. 
+Sets the new user handler for the configuration updates. This handler will be called every time a new configuration is
+created or an old configuration is updated.
 
 The handler must adhere to the following signature:
 ```
 function(newConfiguration, callback)
```

Comment:
Well, it's not missing, really. That's the prototype for the handler function the user have to pass to the `setConfigurationHandler()` function being described in that section. The user can use any name they want.